### ISSUE TITLE
example unattended-installation script in documentation as written waits for user input

### ISSUE THREAD

**Issue by dm-wyncode** [dokku/dokku#2311]
Description of problem:

I created an unattended install script as described [here in the dokku docs](http://dokku.viewdocs.io/dokku/getting-started/install/debian/#unattended-installation).

This is the script I am using: https://gist.github.com/dm-wyncode/109f9134c2f0a3575953361483be559a

I changed 

`sudo apt-get install dokku` 

to 

`sudo apt-get install -qq -y dokku` 

otherwise the script waited for a "Y" input to install.

copied and pasted from dokku docs:
```
# install prerequisites
sudo apt-get update -qq > /dev/null
sudo apt-get install -qq -y apt-transport-https

# install docker
wget -nv -O - https://get.docker.com/ | sh

# install dokku
wget -nv -O - https://packagecloud.io/gpg.key | apt-key add -
echo "deb https://packagecloud.io/dokku/dokku/ubuntu/ trusty main" | sudo tee /etc/apt/sources.list.d/dokku.list
sudo apt-get update -qq > /dev/null
### CHANGED THE SCRIPT HERE OTHERWISE IT WAITED FOR INPUT ###
sudo apt-get install dokku
########
sudo dokku plugin:install-dependencies --core
```

**Comment by josegonzalez** [comment]
Pull requests welcome!

**Comment by dm-wyncode** [comment]
Will do. Thanks for the guidance. I am still new to GitHub participation thing!

**Comment by josegonzalez** [comment]
Generally speaking, if you find our docs obtuse, you have two routes:

- Fix/pull request if you have a fix
- Open an issue to ask for clarification

In both cases we will provide guidance. Our goal is to ensure our docs are the best they can be, and to be as bug-free as possible, so however you can help us there would be *great* :)

Here are our [contributing](https://github.com/dokku/dokku/blob/master/CONTRIBUTING.md) docs in case you have any questions!

**Comment by josegonzalez** [comment]
Closed by #2314
